lib/ansible/modules/cloud/ovirt/ovirt_snapshot.py

```diff
@@ -112,6 +122,10 @@
                   at following url: http://ovirt.github.io/ovirt-engine-api-model/master/#types/snapshot."
     returned: On success if snapshot is found.
     type: dict
+snapshots:
+    description: List of deleted snapshots
```

Comment:
Please also document that it's returned only in case user specify `keep_days_old` parameter.